content/water-data-science-2021.md

```diff
@@ -65,7 +65,7 @@ Advancing machine learning architectures for water prediction {#pgdl}
 {{< figure src="/static/FY21DataSci/pgdl_flubber_small.gif" alt="Process-guided deep learning predictions of stream temperature" class="side-by-side" >}}
 
 <p class="side-by-side" >
-There is swirling mix of excitement, fear, misunderstanding, and raw wonder concerning the role of machine learning in water prediction. In the Data Science branch, we’re excited about the potential use of ML in all kinds of USGS projects. But being part of the USGS, we also understand the positive modeling legacy our agency has, which could be tarnished by misuse or blind adoption of ML. A lot of the fear surrounding ML in environmental sciences comes from a combination of how well it seems to perform in prediction and how effortless applying it to the problem de jour seems to be. There is a lot of nuance to each of these assumptions, as we rarely have enough data to build a truly generalizable environmental ML model, and while these models may be faster to spin up, major challenges exist for dealing with data sparsity, estimating uncertainty, and applying knowledge-based constraints that require custom architectures and cutting-edge research. We have partnerships with the Kumar lab at U-MN, the Jia lab at Pitt, and the Shen lab at Penn State to collaborate on the exciting field of "<a href="https://labs.waterdata.usgs.gov/visualizations/gages-through-the-ages/index.html#/">Knowledge Guided Machine Learning</a>". 
+There is swirling mix of excitement, fear, misunderstanding, and raw wonder concerning the role of machine learning in water prediction. In the Data Science branch, we’re excited about the potential use of ML in all kinds of USGS projects. But being part of the USGS, we also understand the positive modeling legacy our agency has, which could be tarnished by misuse or blind adoption of ML. A lot of the fear surrounding ML in environmental sciences comes from a combination of how well it seems to perform in prediction and how effortless applying it to the problem de jour seems to be. There is a lot of nuance to each of these assumptions, as we rarely have enough data to build a truly generalizable environmental ML model, and while these models may be faster to spin up, major challenges exist for dealing with data sparsity, estimating uncertainty, and applying knowledge-based constraints that require custom architectures and cutting-edge research. We have partnerships with the Kumar lab at U-MN, the Jia lab at Pitt, and the Shen lab at Penn State to collaborate on the exciting field of "<a href="https://sites.google.com/umn.edu/kgml/home">Knowledge Guided Machine Learning</a>". 
```

Comment:
Original: "how effortless applying it to the problem de jour seems to be" can be clarified as:

New: "how seemingly effortless it is to apply ML to the problem du jour"

(also fixed the de => du)